packages/react-email/src/utils/esbuild/renderring-utilities-exporter.ts

```diff
@@ -29,6 +29,7 @@ export const renderingUtilitiesExporter = (emailTemplates: string[]) => ({
         return {
           contents: `${await fs.readFile(pathToFile, 'utf8')};
           export { render } from 'react-email-module-that-will-export-render'
+          export { pretty } from 'react-email-module-that-will-export-pretty'
```

Comment:
we don't need to add another import for this, it can be imported from the same as `render`, like

```suggestion
          export { render, pretty } from 'react-email-module-that-will-export-render'
```

Reply:
updating